cmd/skydive_agent/skydive_agent.go

```diff
@@ -42,12 +42,14 @@ import (
 )
 
 func usage() {
-	fmt.Printf("Usage: %s -conf <config.ini> [-h]\n", os.Args[0])
+	fmt.Printf("Usage: %s -conf <config.ini> -trace <trace.pcap> [-h]\n", os.Args[0])
 }
 
 func main() {
 	filename := flag.String("conf", "/etc/skydive/skydive.ini",
 		"Config file with all the skydive parameter.")
+	pcaptrace := flag.String("trace", "",
```

Comment:
I think this functionality should be moved to another binary, like a tool to read a pcap file sending sflow packet to the agent. BTW I think this feature is really useful for functional testing for instance.